lib/src/model/write_characteristic_info.dart

```diff
@@ -3,6 +3,8 @@ import 'package:flutter_reactive_ble/src/model/qualified_characteristic.dart';
 import 'package:flutter_reactive_ble/src/model/result.dart';
 import 'package:meta/meta.dart';
 
+/// Info containing the [QualifiedCharacteristic] and the result of the
+/// write characteristic operation.
```

Comment:
~"Info containing"? "Info" is way too abstract [to my taste]. I'd consider something like "a class", "a data structure".~

~But even better would be not to repeat the declaration ("A write-operation result."?).~

This is not a public class.